lib/linguist/languages.yml

```diff
@@ -2543,6 +2543,15 @@ MiniD:
   tm_scope: none
   ace_mode: text
   language_id: 231
+MiniZinc:
+  type: programming
+  color: "#1295e6"
+  extensions:
+  - ".mzn"
+  - ".dzn"
```

Comment:
Hm, I think we should put `.dzn` into a separate data language (and apply the same grammar) to avoid skewing the statistics with large data files. Is there a proper name to designate `.dzn` files?